d7-tests/rules.test

```diff
@@ -191,8 +191,8 @@ class RulesTestCase extends DrupalWebTestCase {
       $action->execute();
       $this->fail("Execution hasn't created an exception.");
     }
-    catch (RulesEvaluationException $e) {
-      $this->pass("RulesEvaluationException was thrown: ". $e);
+    catch (EvaluationException $e) {
+      $this->pass("EvaluationException was thrown: ". $e);
```

Comment:
This is the old Drupal 7 testing file and should NOT be modified. It is only there to keep track what we already cover and it is not executed.